scraper.py

```diff
@@ -92,24 +144,23 @@ def make_soup(url, encoding, absolute=False, base='', parser='lxml'):
     return soup
 
 
-def save(url, date, title, explanation, picture_url, data_version):
-    data = {
-        'url': url,
-        'date': date,
-        'title': title,
-        'explanation': explanation,
-        'picture_url': picture_url,
-    }
-
-    version_data = {
-        'url': url,
-        'data_version': data_version,
-    }
+def save(url, date, title, credit, explanation, picture_thumbnail_url, picture_url, video_url, data_version):
+    data = OrderedDict()
```

Comment:
This is probably just my unfamiliarity with morph.io, but what are some of the advantages of OrderedDict in this case? Another comment mentioned that the data was out of order. In what way? I understand that Python dictionaries don't have a predictable order; does that affect the resulting database as well? Again, probably just my confusion. :smile: